frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -100,27 +100,12 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
       ),
     );
 
-  const cleanFormData = () => {
```

Comment:
We already have checks in place to avoid `///////` inputs (https://github.com/opendatahub-io/odh-dashboard/pull/1512), so considered removing this